src/include/migraphx/rewrite_dequantizelinear.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef MIGRAPHX_GUARD_RTGLIB_REWRITE_DEQUANTIZELINEAR_HPP
+#define MIGRAPHX_GUARD_RTGLIB_REWRITE_DEQUANTIZELINEAR_HPP
+
+#include <string>
+#include <migraphx/config.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+
+struct module;
+
+/**
+ * Rewrite dequantizelinear to equivalent operators
+ */
+struct rewrite_dequantizelinear
+{
+    std::string name() const { return "rewrite_dequantizelinear"; }
+    void apply(module& p) const;
```

Comment:
we had better use module& m, the name "p" here is we use a program here before, but we did not change it to "m" when we change the program to module.

Reply:
Fixed